src/ape/api/providers.py

```diff
@@ -1100,7 +1100,7 @@ def _prepare_call(self, txn: TransactionAPI, **kwargs) -> List:
         txn_dict.pop("maxFeePerGas", None)
         txn_dict.pop("maxPriorityFeePerGas", None)
 
-        block_identifier = kwargs.pop("block_identifier", kwargs.pop("block_id", "latest"))
```

Comment:
What's wrong with `"latest"`?

Reply:
nothing, besudes it's the default already when passing `None` to web3, so less management on our end by using `None`.